internal/tui/tui_test.go

```diff
@@ -0,0 +1,97 @@
+package tui
+
+import (
+	"errors"
+	"testing"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/shazow/wifitui/wifi"
+	"github.com/shazow/wifitui/wifi/mock"
+	"github.com/stretchr/testify/require"
```

Comment:
Can we avoid adding a new dependency for now? Let's just use the latest idiomatic practices of the `testing` stdlib.